RELEASING/README.md

```diff
@@ -64,13 +64,21 @@ that belong to the MAJOR.MINOR version.
 The MAJOR.MINOR branch is normally a "cut" from a specific point in time from the master branch.
 Then (if needed) apply all cherries that will make the PATCH
 
+Next update the `CHANGELOG.md` with all the changes that are included in the release. Make sure you have
+set your GITHUB_TOKEN environment variable.
+
+```bash
+# will overwrites the local CHANGELOG.md, somehow you need to merge it in
+github-changes -o apache -r incubator-superset --token $GITHUB_TOKEN --between-tags <PREVIOUS_RELEASE_TAG>...<CURRENT_RELEASE_TAG>
```

Comment:
Totally, I used a mix of `git log` and `github-changes`. Were you able to get the desired output with `git`?

Reply:
Yeah, the `git` output made total sense, but `github-changes` was essentially useless. Since the changelog was quite short, I essentially built it by hand as I didn't want to get hung up on it, but made a mental note try to fix this later.